modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,148 @@
+        },
+        texts:{
+            dropdownName: 'Creditserweiterung',
+            creditsOverview: 'Credits-Übersicht',
+            coinsProtokoll: 'Coinsprotokoll',
+            earnedCredits: 'Gesamtcredits',
+            creditsToNextRank: 'Credits zum nächsten Dienstgrad',
+            updateMessage: 'Werte aktualisieren',
+            allianceFunds: 'Verbandskasse',
+            noFurtherRank: 'Keine Höheren Ränge'
+        }
+    };
+    I18n.translations.en.lssm.creditserweiterung = {
+        earnedCreditsRegEx: /(.*)([0-9],)? earned Credits.*/i,
+        ranks: {
+            0: 'Probie',
+            200: 'Firefighter',
+            10000: 'Senior Firefighter',
+            100000: 'Fire Apparatus Operator',
+            1000000: 'Lieutenant',
+            5000000: 'Captain',
+            20000000: 'Staff Captain',
+            50000000: 'Battalion Chief',
+            1000000000: 'Division Chief',
+            2000000000: 'Deputy Chief',
+            5000000000: 'Fire Chief'
+        },
+        texts:{
+            reloadMessage: 'Please open your profile<br>and reload the website!',
+            dropdownName: 'Creditsextension',
+            creditsOverview: 'Overview',
+            coinsProtokoll: 'Coins List',
+            earnedCredits: 'Totally earned Credits',
+            creditsToNextRank: 'Credits to next rank',
+            updateMessage: 'Values are updated<br>all 5 minutes.',
+            allianceFunds: 'Alliance Funds',
+            noFurtherRank: 'There are no further ranks'
+        }
+    };
+    I18n.translations.nl.lssm.creditserweiterung = {
+        earnedCreditsRegEx: /(.*)([0-9]\.)? verdiende Credits.*/i,
+        ranks: {
+            0: 'Aspirant',
+            200: 'Brandwacht (Manschap A)',
+            10000: 'Hoofdbrandwacht (Manschap B)',
+            100000: 'Brandmeester',
+            1000000: 'Hoofdbrandmeester',
+            5000000: 'Commandeur',
+            50000000: 'Adjunct-Hoofdcommandeur',
+            500000000: 'Hoofdcommandeur',
+            5000000000: 'Erelid'
+        },
+        texts:{
+            reloadMessage: 'Open uw profiel<br>open en de website herladen!',
+            dropdownName: 'Kredietexpansie',
+            creditsOverview: 'Overzicht statistiek',
+            coinsProtokoll: 'Coin protocol',
+            earnedCredits: 'Totale studiepunten',
+            creditsToNextRank: 'Credits voor de volgende rang',
+            updateMessage: 'Waarden actualiseren<br>om de 5 minuten.',
+            allianceFunds: 'Teamkas',
+            noFurtherRank: 'Geen verdere rangen meer'
+        }
+    };
+
+    function getCreditsOfNextRank(earnedCredits){
+        let ranks = I18n.t('lssm.creditserweiterung.ranks');
+        for (var key in ranks) {
+            if(earnedCredits > key){
+                continue;
+            } else {
+                return key;
+            }
+        }
+        return I18n.t('lssm.creditserweiterung.noFurtherRank');;
```

Comment:
Dieser Rückgabewert wird in Zeile 120 ein Fehler erzeugen, da hier eine mathematische Operation darauf folgt.
Gib hier null zurück und behandle den Fall dann mit einer IF Bedingung ab Zeile 119.